ops/terraform/main.tf

```diff
@@ -98,26 +84,17 @@ module "ocr_autoscale" {
   weekend_capacity_instances = 1
 }
 
-# module "compute" {
-#   source         = "./modules/container_instances"
-#   location       = data.azurerm_resource_group.rg.location
-#   resource_group = data.azurerm_resource_group.rg.name
-#   environment    = local.environment
-#   app_subnet     = module.networking.appsubnet_id
-#   # web_subnet_id   = module.networking.websubnet_id
-#   # app_subnet_id   = module.networking.appsubnet_id
-#   # web_host_name   = local.app.web_host_name
-#   # web_username    = local.app.web_username
-#   # web_os_password = local.app.web_os_password
-#   # app_host_name   = local.app.app_host_name
-#   # app_username    = local.app.app_username
-#   # app_os_password = local.app.app_os_password
-# }
-
-##########
-## 04-config
-##########
+module "database" {
+  source              = "./modules/database"
+  resource_group_name = data.azurerm_resource_group.rg.name
+  subnet              = module.networking.dbsubnet_id
+}
 
-##########
-## 07-Monitor
-##########
\ No newline at end of file
+module "vault" {
+  source              = "./modules/vault"
+  resource_group_name = data.azurerm_resource_group.rg.name
+  azure_tenant_id     = var.azure_tenant_id
+  object_id           = var.object_id
+  vite_api_url        = var.vite_api_url
```

Comment:
We need to make sure this is not the endpoint the frontend is also using(because vite is a frontend tool). this is where we might need to create a private endpoint from backend api to the db.

Reply:
Done.

Reply:
Refactored with `postgresql_flexible_server` with a `delegated_subnet_id` and `private_dns_zone_id`